### ISSUE TITLE
Gitea dump fails due to "Packages isn't enabled"

### ISSUE THREAD

**Issue by duchenpaul** [go-gitea/gitea#23709]
### Description

My gitea is running with packages function disabled, in recently upgrade, I found the dump is not working  due to below error:
```
04:34:03  2023-03-26 04:34:03 AM: Dumping gitea...
04:34:04  2023/03/26 04:34:04 ...g/config_provider.go:30:deprecatedSetting() [E] Deprecated fallback `[server]` `LFS_CONTENT_PATH` present. Use `[lfs]` `PATH` instead. This fallback will be/has been removed in v1.19.0
04:34:04  2023/03/26 04:34:04 ...dules/setting/log.go:331:initLogFrom() [I] Gitea Log Mode: Console(Console:info)
04:34:04  2023/03/26 04:34:04 ...dules/setting/log.go:286:generateNamedLogger() [I] Router Log: Console(console:info)
04:34:04  2023/03/26 04:34:04 ...les/setting/cache.go:75:loadCacheFrom() [I] Cache Service Enabled
04:34:04  2023/03/26 04:34:04 ...les/setting/cache.go:90:loadCacheFrom() [I] Last Commit Cache Service Enabled
04:34:04  2023/03/26 04:34:04 ...s/setting/session.go:74:loadSessionFrom() [I] Session Service Enabled
04:34:04  2023/03/26 04:34:04 ...g/config_provider.go:30:deprecatedSetting() [E] Deprecated fallback `[mailer]` `HOST` present. Use `[mailer]` `SMTP_ADDR` instead. This fallback will be/has been removed in v1.19.0
04:34:04  2023/03/26 04:34:04 ...es/setting/mailer.go:202:loadMailerFrom() [W] connecting over insecure SMTP protocol to non-local address is not recommended
04:34:04  2023/03/26 04:34:04 ...es/setting/mailer.go:236:loadMailerFrom() [I] Mail Service Enabled
04:34:04  2023/03/26 04:34:04 ...es/setting/mailer.go:258:loadNotifyMailFrom() [I] Notify Mail Service Enabled
04:34:04  2023/03/26 04:34:04 ...s/storage/storage.go:175:initAttachments() [I] Initialising Attachment storage with type: 
04:34:04  2023/03/26 04:34:04 ...les/storage/local.go:45:NewLocalStorage() [I] Creating new Local Storage at /data/gitea/attachments
04:34:04  2023/03/26 04:34:04 ...s/storage/storage.go:165:initAvatars() [I] Initialising Avatar storage with type: 
04:34:04  2023/03/26 04:34:04 ...les/storage/local.go:45:NewLocalStorage() [I] Creating new Local Storage at /data/gitea/avatars
04:34:04  2023/03/26 04:34:04 ...s/storage/storage.go:191:initRepoAvatars() [I] Initialising Repository Avatar storage with type: 
04:34:04  2023/03/26 04:34:04 ...les/storage/local.go:45:NewLocalStorage() [I] Creating new Local Storage at /data/gitea/repo-avatars
04:34:04  2023/03/26 04:34:04 ...s/storage/storage.go:185:initLFS() [I] Initialising LFS storage with type: 
04:34:04  2023/03/26 04:34:04 ...les/storage/local.go:45:NewLocalStorage() [I] Creating new Local Storage at /data/git/lfs
04:34:04  2023/03/26 04:34:04 ...s/storage/storage.go:197:initRepoArchives() [I] Initialising Repository Archive storage with type: 
04:34:04  2023/03/26 04:34:04 ...les/storage/local.go:45:NewLocalStorage() [I] Creating new Local Storage at /data/gitea/repo-archive
04:34:04  2023/03/26 04:34:04 cmd/dump.go:246:runDump() [I] Dumping local repositories... /data/git/repositories
04:34:08  2023/03/26 04:34:08 cmd/dump.go:285:runDump() [I] Dumping database...
04:34:11  2023/03/26 04:34:11 cmd/dump.go:297:runDump() [I] Adding custom configuration file from /data/gitea/conf/app.ini
04:34:11  2023/03/26 04:34:11 cmd/dump.go:313:runDump() [I] Custom dir /data/gitea is inside data dir /data/gitea, skipped
04:34:11  2023/03/26 04:34:11 cmd/dump.go:325:runDump() [I] Packing data directory.../data/gitea
04:34:12  Failed to dump packages: Packages isn't enabled
04:34:12  2023/03/26 04:34:11 cmd/dump.go:162:fatal() [F] Failed to dump packages: Packages isn't enabled
04:34:12  Failed! Return Status = 1
```

Here is my dump command:
```bash
gitea dump -c /data/gitea/conf/app.ini
```

Config:
```
[packages]
ENABLED = false
```

### Gitea Version

1.19.0

### Can you reproduce the bug on the Gitea demo site?

No

### How are you running Gitea?

In docker

### Database

MySQL

**Comment by duchenpaul** [comment]
I am seeking if there were parameters I can pass into dump command in order to skip the package backup, unfortunately I did not see in doc

**Comment by wxiaoguang** [comment]
It could be fixed by #23631  (no backport yet)

**Comment by Zettat123** [comment]
> It could be fixed by https://github.com/go-gitea/gitea/pull/23631 (no backport yet)

Looks like the auto backport failed. I'll make the backport manually.

**Comment by Zettat123** [comment]
I closed this issue since the backport PR(#23730) has been merged. Feel free to reopen if you need more help.
